sdk/identity/Azure.Identity/src/AzurePowerShellCredential.cs

```diff
@@ -253,7 +253,7 @@ private static AccessToken DeserializeOutput(string output)
                         break;
 
                     case "ExpiresOn":
-                        expiresOn = DateTimeOffset.Parse(e.Value, CultureInfo.InvariantCulture).ToUniversalTime();
+                        expiresOn = DateTimeOffset.Parse(e.Value, CultureInfo.CurrentCulture).ToUniversalTime();
```

Comment:
Should we maybe include a test using the case from the linked bug?

Reply:
added